src/frontend/webpack.config.js

```diff
@@ -44,4 +67,15 @@ module.exports = {
       { enforce: 'pre', test: /\.js$/, loader: 'source-map-loader' },
     ],
   },
+
+  plugins: [
+    // Use module replacement to override any number of Richie components as defined in the settings
+    ...Object.entries(overrides).map(
+      entry =>
+        new webpack.NormalModuleReplacementPlugin(
+          new RegExp(`components\/${entry[0]}$`),
```

Comment:
Did you did that to force only component overriding ?

Reply:
Yes! This is the result of a discussion with @sampaccoud, and is also why we moved to absolute paths. It makes it way easier for dependents to override components